### PR TITLE
added getVolume

### PR DESCRIPTION
**Pull request by SpeckyYT**

### CODE REVIEW

**Comment by SpeckyYT** [comment]
I'll update the queues anyways

File: src/Player.js [hunk 1]
```diff
@@ -291,6 +291,19 @@ class Player {
         queue.dispatcher.setVolumeLogarithmic(percent / 200);
     }
 
+    /**
+     * Gets the volume.
+     * @param {string} guildID 
+     */
+    getVolume(guildID) {
+        // Gets guild queue
+        let queue = this.getQueue(guildID);
```
Comment (SushiBtw) [inline src/Player.js:300]: Please change to `this.queues.get()` ty <3

**Review by SushiBtw — CHANGES_REQUESTED** [review]
change to maping, please!

**Review by SushiBtw — COMMENTED** [review]
thanks!

**Review by SushiBtw — APPROVED** [review]

**Pull request merged**